systemtests/cmake/BareosSystemtestFunctions.cmake

```
@@ -24,7 +24,7 @@ macro(create_systemtests_directory)
   configurefilestosystemtest("systemtests" "scripts" "functions" @ONLY "")
   configurefilestosystemtest("systemtests" "scripts" "cleanup" @ONLY "")
   configurefilestosystemtest("systemtests" "scripts" "redirect_output" @ONLY "")
-  configurefilestosystemtest("systemtests" "scripts" "mysql.sh" @ONLY "")
+  configurefilestosystemtest("systemtests" "scripts" "mariadb.sh" @ONLY "")
```

Your remark make sense. I will keep the mysql.sh, and check if some "non mysql" related test are still using that one.
```suggestion
  configurefilestosystemtest("systemtests" "scripts" "mysql.sh" @ONLY "")
  configurefilestosystemtest("systemtests" "scripts" "mariadb.sh" @ONLY "")
```